vk_clone/vk_clone/Network/VKAPIFriendsAPIService.swift

```diff
@@ -0,0 +1,48 @@
+// VKAPIFriendsAPIService.swift
+// Copyright © RM. All rights reserved.
+
+import Alamofire
+import Foundation
+import PromiseKit
+import SwiftyJSON
+
+final class VKAPIFriendsAPIService {
```

Comment:
```suggestion
final class VKAPIFriendsAPIService {
```
многова-то API в названии....

Reply:
согласен)